src/test/java/project/eepw/softwaretestingcrud/domain/DomainUtils.java

```diff
@@ -0,0 +1,31 @@
+package project.eepw.softwaretestingcrud.domain;
+
+import project.eepw.softwaretestingcrud.domain.post.entity.Post;
+import project.eepw.softwaretestingcrud.domain.user.entity.User;
+
+import java.util.Collections;
+
+public final class DomainUtils {
+    private DomainUtils() {
+
+    }
+
+    public static Post makePost() {
+        return Post.builder()
+                .id(1L)
+                .content("Some random post content")
```

Comment:
I do not know actually if it it is good idea to create methods like this in the seperate class